index.html

```diff
@@ -1,152 +1,157 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <link rel="stylesheet" href="style.css" />
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
-  </head>
-  <body>
-    <div class="content">
-      <header class="site-header">
-        <div class="site-wrapper">header</div>
-      </header>
-      <main class="site-main">
-        <section class="first-section">
-          <h2>Heading</h2>
-          <div class="site-wrapper">
-            <p>text</p>
-          </div>
+
+<head>
+  <link rel="stylesheet" href="./style.css/main.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet" />
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio</title>
+</head>
+<body>
+  <div class="content">
+    <header class="site-header">
+      <div class="site-wrapper">
```

Comment:
Where is the closing tag?

Reply:
Closing tag of wich one?
I checked it and all of these 3 tags has closing tag,am I wrong?

Reply:
Maybe closing tag for site wrapper.